src/System.Windows.Forms/src/System/Windows/Forms/AxHost.AxPropertyTypeEditor.cs

```diff
@@ -30,21 +28,21 @@ public AxPropertyTypeEditor(AxPropertyDescriptor pd, Guid guid)
             ///  the user to modify the value.  Host assistance in presenting UI to the user
             ///  can be found through the valueAccess.getService function.
             /// </summary>
-            public override object EditValue(ITypeDescriptorContext context, IServiceProvider provider, object value)
+            public override object? EditValue(ITypeDescriptorContext? context, IServiceProvider provider, object? value)
             {
                 try
                 {
-                    object instance = context.Instance;
+                    object? instance = context?.Instance;
                     _propertyDescriptor._owner.ShowPropertyPageForDispid(_propertyDescriptor.Dispid, _guid);
                 }
-                catch (Exception ex1)
+                catch (Exception ex)
                 {
                     if (provider is not null)
                     {
-                        IUIService uiSvc = (IUIService)provider.GetService(typeof(IUIService));
+                        IUIService? uiSvc = (IUIService?)provider.GetService(typeof(IUIService));
```

Comment:
```suggestion
                        IUIService? uiSvc = (IUIService?)provider?.GetService(typeof(IUIService));
```
...and remove the check above?